src/libponyc/pass/docgen.c

```diff
@@ -852,6 +852,11 @@ static doc_sources_t* copy_source_to_doc_src(docgen_t* docgen, source_t* source,
     fprintf(file, "hide:\n");
     fprintf(file, "  - toc\n");
     fprintf(file, "---\n");
+    // tell the mkdocs theme not index source files for search
+    fprintf(file, "---\n");
```

Comment:
I think there is only one frontmatter in a yaml file and the search section should be part of it, no?
My suggestion is, to put the `search` bits right below  the `- toc` bits.

Reply:
ah yes.

Reply:
updated.